objects/Fan-MadeAccessories.aa8b38/CleanUpHelper.26cf4b.ttslua

```diff
@@ -209,10 +208,10 @@ function resetCounters()
   end
 
   -- reset doom on agenda
-  -- local doomcounter = getObjectFromGUID("85c4c6")
-  -- if doomcounter ~= nil then
-  --   doomcounter.call("setToZero")
-  -- end
+  local doomcounter = getObjectFromGUID("85c4c6")
+  if doomcounter ~= nil then
+    doomcounter.call("updateVal")
```

Comment:
Nit: Even if 0 ends up as the default for a nil, should pass it here for clarity